.gitignore

```diff
@@ -292,6 +292,9 @@ docs/_build/
 # pyenv
 .python-version
 
+# pyright
+pyrightconfig.json
```

Comment:
Curious: what is pyrightconfig?

Reply:
its my config file for showcasing the virtual env which talks to the LSP server through my neovim.